docs/docusaurus/docs/cloud/get_support.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: 'Get GX Cloud support'
+title: 'Get GX Cloud support'
+description: Learn what resources are available to help you resolve issues with GX Cloud.
+---
+
+The information provided here is intended to help you resolve GX Cloud issues. Several resources are available, including the collective knowledge of a very active community and experienced GX Developer Advocates (DAs). Knowing what resources are available and how to ask questions will simplify your support experience.
+
+## Try solving your issue first
+
+The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your own knowledge and provides you with an opportunity to share that knowledge with the community. 
+
+### Search the GX Cloud documentation
+
+Search the docs you're using currently for an answer to your issue or question. If you're new to GX Cloud and are using Snowflake, try the [Quickstart for GX Cloud and Snowflake](quickstarts/snowflake_quickstart.md). If you're interested in reviewing features and functionality, see [Set up GX Cloud](/docs/cloud/set_up_gx_cloud).  
+
+### Search with your favorite browser
```

Comment:
What do we say "browser" instead of "search engine"?

Reply:
@joshzzheng a browser is a piece of software that retrieves and displays web pages; a search engine is a website that helps people find web pages from other websites. For example, Firefox and Safari are web browsers, not search engines. The shorter form, _browser_ is the correct choice and is the term used by technical writers. See also _[Supported browsers](https://docs.getdbt.com/docs/cloud/about-cloud/browsers)_ in the dbt docs

Reply:
But it's the search engine that has the impact on your search experience, not the browser. dbt actually uses search engine: https://docs.getdbt.com/community/resources/getting-help

Reply:
@joshzzheng _browser_ is the correct term, and what I've used at all previous employers.

Reply:
Sure. We should probably say "browser's search engine" though.